app/test_main.py

```diff
@@ -1,3 +1,120 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, expected_converted_ages",
+        [
+            pytest.param(
+                0,
+                0,
+                [0, 0],
+                id="should return zeroes if animal\'s age equal zeroes"
+            ),
+            pytest.param(
+                14,
+                14,
+                [0, 0],
+                id="should return zeroes if animal\'s age less than 15"
+            ),
+            pytest.param(
+                15,
+                15,
+                [1, 1],
+                id="should calculate first human year "
+                   "if animal\'s age equal 15"
+            ),
+            pytest.param(
+                23,
+                23,
+                [1, 1],
+                id="should calculate first human year "
+                   "if animal\'s age are between 15 and 24"
+            ),
+            pytest.param(
+                24,
+                24,
+                [2, 2],
+                id="should calculate second human year "
+                   "if animal\'s age equal 24"
+            ),
+            pytest.param(
+                27,
+                27,
+                [2, 2],
+                id="should calculate second human year "
+                   "if animal\'s age are between 24 and 28"
+            ),
+            pytest.param(
+                28,
+                28,
+                [3, 2],
+                id="should increase cat age to third human year after 28"
+            ),
+            pytest.param(
+                26,
+                29,
+                [2, 3],
+                id="should increase dog age to third human year after 29"
+            ),
+            pytest.param(
+                100,
+                100,
+                [21, 17],
+                id="should accurately convert large cat and dog ages"
+            ),
+            pytest.param(
+                10000,
+                10000,
+                [2496, 1997],
+                id="should accurately convert extremely large cat and dog ages"
+            ),
+            pytest.param(
+                -1,
+                -1,
+                [0, 0],
+                id="should return zeroes if animal\'s age is negative"
+            ),
+            pytest.param(
+                10000,
+                10000,
+                [2496, 1997],
+                id="should accurately convert float cat and dog ages"
```

Comment:
The test case with id 'should accurately convert float cat and dog ages' is identical to the previous test case for large ages. This might be a mistake, as it does not test float values. Consider revising this test case to use actual float inputs if that is the intended purpose.